ports/esp32/mpconfigport.h

```diff
@@ -201,6 +201,61 @@
 
 #define MP_STATE_PORT MP_STATE_VM
 
+#ifndef MICROPY_HW_ENABLE_USBDEV
+#define MICROPY_HW_ENABLE_USBDEV            (SOC_USB_OTG_SUPPORTED)
+#endif
+
+#if MICROPY_HW_ENABLE_USBDEV
+#define MICROPY_SCHEDULER_STATIC_NODES        (1)
+#define MICROPY_HW_USB_CDC_DTR_RTS_BOOTLOADER (1)
+
+// Enable USB-CDC serial port
+#ifndef MICROPY_HW_USB_CDC
+#define MICROPY_HW_USB_CDC                  (1)
```

Comment:
There's a second definition of this on L319 that should probably be merged with this one, somehow.

(maybe add it around line 207, and define it to the value of MICROPY_HW_ENABLE_USBDEV if otherwise undefined?)

Reply:
Yeah thanks, I'd missed this duplication. I've consolidated this a bit better now, I think it's fairly clear.